authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from .views import (
+    RetrieveUpdateProfileView,
+    ListProfileView)
+
+urlpatterns = [
+    # path('profiles/<str:username>/', UpdateProfileView.as_view()),
```

Comment:
@NLSanyu Remove the commented code here. Anything that is not used should not be pushed.